exercises/kindergarten-garden/description.md

```diff
@@ -7,13 +7,13 @@ actual dirt, and grow actual plants.
 
 They've chosen to grow grass, clover, radishes, and violets.
 
-To this end, they've put little styrofoam cups along the window sills,
+To this end, they've put little Styrofoam cups along the window sills,
```

Comment:
I'd rather see a generic term used here. plastic? paper?
Can we just omit the word Styrofoam?
And does it matter they are little?

And since I'd grow a plant in a pot, how about a simply:
"they've put pots along the window sills"

Reply:
Plastic cups should suffice.  I'll make the change.

Reply:
Whatever we do decide here, we should then change other occurences of “cups” as well.